### DISCUSSION TITLE
Multiple strategies running on one symbol: Qs and advice req

### DISCUSSION THREAD

**Discussion by fredmonroe** [nautechsystems/nautilus_trader#1676] [Q&A]
Hi, this is partly to check my understanding from what i'm seeing in testing and partly asking for advice 

Say  we have multiple strategies trading on a single node - and they are all trading the same symbol...
It seems like if they make decisions off of something like self.portoflio.net_position() - they will end up interfering with each other
(in other words, the portfolio is shared acrosss strategies)

This could be either good or bad depending on what you're trying to do i imagine.

But suppose I want each strategy to track its own position independently?

I see that the position objects in the cache are labelled with which strategy they belong to.
I see some of the example strategies seem to track their position with a local variable like _pos

I'm wondering for longer term strategies where the node might be going up and down what the recommended way to approach this is?

one approach i see is:
on strategy start - potentially query the positions/fills for just my strategy to sync up my internal position state and then track it myself from there as fills come into my strategy

any advice on how to run multiple *independent* strategies on one symbol across longer time horizons is appreciated

thanks

**Comment by OnlyC** [comment]
If you trading cryptos, I suggest that you create multiple child accounts for each strategy.

**Comment by cjdsellers** [comment]
Hey @fredmonroe

I'd suggest going through [this doc](https://nautilustrader.io/docs/nightly/concepts/live). In particular, `filter_position_reports` and `external_order_claims` can help here.
